 Python 3
Test exceptions
